rts/at/integration/pom.xml

```diff
@@ -133,6 +133,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
```

Comment:
we need this dependency version. It would be better to add it in the bom I think.